aptos-move/framework/aptos-framework/sources/SimpleMap.move

```diff
@@ -133,6 +134,15 @@ module AptosFramework::SimpleMap {
         }
     }
 
+    public fun get_entry<Key: store, Value: store>(
+        map: &SimpleMap<Key, Value>,
+        index: u64,
+    ) : (&Key, &Value) {
+        assert!(index < Vector::length(&map.data) && index >= 0, Errors::invalid_argument(EINDEX_OUT_OF_BOUNDS));
+
+        (&Vector::borrow(&map.data, index).key, &Vector::borrow(&map.data, index).value)
+    }
```

Comment:
Interesting, so this is getting the index in the map.  So, this is basically to iterate over the entire map?  Maybe we should do a "iter()" like rust, or something that returns a reference to the entire underlying vector.  Though this is not required for this right now.